views/index.pug

```diff
@@ -0,0 +1,63 @@
+html
+  include partials/head
+  body
+    div.content
+      h1.title Ballantine Adult Fantasy
+      img(src="/img/unicorncolophon.jpg", alt="Ballantine Adult Fantasy - Unicorn Colophon")
+      h2 Documentation for API
+      div.description
+        p Instructions as to how to make proper use of the BAF API.
+        p Quick reference outside of the technical details:
+        p The series ran from 1969-1974 and was edited by Lin Carter.
+          |  There are 18 books "pre-series" that are spiritually if not
+          | technically published under the Ballantine Adult Fantasy label,
+          | but many of these were later reprinted and emblazoned with
+          | the official unicorn colophon. In the series proper, there
+          | were a total of 65 books published,
+          | and two "post-series", for a grand total of 85 all told.
+        p For the purposes of this API, only those first 18 are currently documented.
+      ul
+        li.item GET /books
+        ul
+          li Route: localhost:8080
+            a(href="/books") /books
+          li Description: Returns a list of all documented books.
+          br
+        li.item GET /books/{id or title}
+        ul
+          li Route: localhost:8080/books/{id or title}
+          li Description: Returns a book matching the title.
+          li Example:
+            a(href="/books/unicorn") /books/unicorn
+          br
+        li.item GET /authors/{id or name}
+        ul
+          li Route: localhost:8080/authors/{id or name}
+          li Description: Returns the matching author's books
+          li Example:
+            a(href="/authors/beagle") /authors/beagle
+          br
```

Comment:
Wouldn't be possible to call either of these by id since the list routes don't include the id in the data returned so the user would never know what the id is